ocs_ci/ocs/longevity_stage2.py

```diff
@@ -0,0 +1,588 @@
+
+
+def measure_pod_creation_time(namespace, num_of_pods):
+    """
+    Measures and Logs the POD Creation Time of all the PODs.
+
+    Args:
+        namespace (str) : Namespace in which the PODs are created.
+        num_of_pods (int) : Number of PODs created.
+
+    Raises:
+        PerformanceException : Raises an exception if POD creation time is greater than the accepted time.
+
+    Logs:
+        POD Creation Time of all the PODs.
+
+    """
+    logs = performance_lib.run_oc_command(
+        "get events --sort-by='{.lastTimestamp}'",
+        namespace,
+    )
+
+    scheduled_time = None
+    pod_no = num_of_pods
+    accepted_creation_time = 12
+
+    for line in logs:
+        logger.info(line)
+        if "Scheduled" in line:
+            scheduled_time = int(line.split()[0][:-1])
+        elif "Created" in line:
+            created_time = int(line.split()[0][:-1])
+            creation_time = scheduled_time - created_time
+            logger.info(f"POD number {pod_no} was created in {creation_time} seconds.")
+            if creation_time > accepted_creation_time:
+                raise ex.PerformanceException(
+                    f"POD creation time is {creation_time} and is greater than "
+                    f"{accepted_creation_time} seconds."
+                )
+            pod_no -= 1
+
+
+def measure_pvc_creation_time(interface, pvc_objs, start_time):
+    """
+    Measures and Logs PVC Creation Time of all PVCs.
+
+    Args:
+        interface (str) : an interface (RBD or CephFS) to run on.
+        pvc_objs (list) : List of PVC objects for which we have to measure the time.
+        start_time (str) : Formatted time from which and on to search the relevant logs.
+
+    Raises:
+        PerformanceException : Raises an exception if PVC creation time is greater than the accepted time.
+
+    Logs:
+        PVC Creation Time of all the PVCs.
+
+    """
+    accepted_creation_time = 1
+    for pvc_obj in pvc_objs:
+        creation_time = performance_lib.measure_pvc_creation_time(
+            interface, pvc_obj.name, start_time
+        )
+
+        logger.info(f"PVC {pvc_obj.name} was created in {creation_time} seconds.")
+        if creation_time > accepted_creation_time:
+            raise ex.PerformanceException(
+                f"PVC {pvc_obj.name} creation time is {creation_time} and is greater than "
+                f"{accepted_creation_time} seconds."
+            )
+
+
+def measure_pvc_deletion_time(interface, pvc_objs):
+    """
+    Measures and Logs PVC Deletion Time of all PVCs.
+
+    Args:
+        interface (str) : an interface (RBD or CephFS) to run on.
+        pvc_objs (list) : List of PVC objects for which we have to measure the time.
+
+    Raises:
+        PerformanceException : Raises an exception if PVC deletion time is greater than the accepted time.
+
+    Logs:
+        PVC Creation Time of all the PVCs.
+
+    """
+    accepted_deletion_time = 2 if interface == constants.CEPHFILESYSTEM else 1
```

Comment:
@ramkiperiy @Avilir  Can you please check and confirm if we are using the correct accepted time here. 
@malhotra700 Triggered a job running stage2 which does sequential PVC deletes and the test failed as the deletion took approx 13 seconds

Reply:
@malhotra700 Please share the test run Jenkin job details here